src/common/safe_io.c

```diff
@@ -119,7 +119,7 @@ ssize_t safe_pwrite(int fd, const void *buf, size_t count, off_t offset)
 }
 
 #ifdef CEPH_HAVE_SPLICE
-ssize_t safe_splice(int fd_in, loff_t *off_in, int fd_out, loff_t *off_out,
+ssize_t safe_splice(int fd_in, off_t *off_in, int fd_out, off_t *off_out,
```

Comment:
`loff_t` is a linux specific type. and is used by underlying splice(2) system call. yeah, it's a 64bit type. but i don't get what you mean by

> 64-bit behavior on glibc is enforced with __USE_FILE_OFFSET64 compiler error.